lib/user_agent/browsers/base.rb

```diff
@@ -3,6 +3,37 @@ module Browsers
     class Base < Array
       include Comparable
 
+      ANDROID     = 'Android'
+      APPLE_WATCH = 'Apple Watch'
+      DALVIK      = 'Dalvik'
+      DARWIN      = 'Darwin'
+      IOS         = 'iOS'
+      IPAD        = 'iPad'
+      IPHONE      = 'iPhone'
+      IPODTOUCH   = 'iPod_touch'
+      IPOD_TOUCH  = 'iPod touch'
+      MACINTOSH   = 'Macintosh'
+      MAC_OS      = 'macOS'
```

Comment:
alphabetize

Reply:
They're alphabetized on the constant name.

Reply:
I lied, sorting it again LINUX wasn't in the right order, fixed now.